Makefile

```diff
@@ -12,4 +12,5 @@ lint:
 	pipenv check ./secure_message ./tests
 
 test: lint
-	pipenv run python run_tests.py
+	pipenv run behave --format progress
+	export APP_SETTINGS=TestConfig && pipenv run python -m unittest && unset APP_SETTINGS
```

Comment:
I think we could probably change this to just `pipenv run pytest` I think the run_tests.py file is a bit bloated especially when this does the same thing?

Reply:
Forgot pytest even existed... thanks ^^